nexus-repository-p2-it/src/test/java/org/sonatype/nexus/repository/p2/internal/P2ProxyIT.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.p2.internal;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+
+import org.sonatype.goodies.httpfixture.server.fluent.Behaviours;
+import org.sonatype.goodies.httpfixture.server.fluent.Server;
+import org.sonatype.nexus.pax.exam.NexusPaxExamSupport;
+import org.sonatype.nexus.repository.Repository;
+import org.sonatype.nexus.repository.http.HttpStatus;
+import org.sonatype.nexus.repository.storage.Asset;
+import org.sonatype.nexus.repository.storage.Component;
+import org.sonatype.nexus.testsuite.testsupport.NexusITSupport;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.sonatype.nexus.testsuite.testsupport.FormatClientSupport.status;
+
```

Comment:
looks like one extra line

Reply:
in: https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/17/commits/94912ebe5ebec6d38ce4e6c99a076d6a7b89a042